engines/bops_api/lib/bops_api/schemas.rb

```diff
@@ -22,6 +22,14 @@ def find!(name, version: nil, schema: nil)
         raise BopsApi::Errors::SchemaNotFoundError, "Unable to find schema '#{schema}'"
       end
 
+      if Rails.env.test?
```

Comment:
I'm not sure about defining this only in a test environment. Is it just to give access to the private method from the test helper?

I feel like we'd lose nothing by removing the conditional and having it defined in all environments.

Reply:
Being removed - https://github.com/unboxed/bops/pull/1805#discussion_r1604985590